tests/phpunit/tests/post/types.php

```diff
@@ -589,4 +589,34 @@ public function test_get_post_types_by_support_excluding_features() {
 	public function test_get_post_types_by_support_non_existant_feature() {
 		$this->assertSameSets( array(), get_post_types_by_support( 'somefeature' ) );
 	}
+
+	/**
+	 * @group oembed
+	 * @ticket 35567
+	 */
+	public function test_register_post_type_is_embeddable_defaults_to_public_argument() {
+		$post_type = register_post_type( rand_str( 10 ) );
```

Comment:
Using randomness in tests is never great. No need for that here.

```suggestion
		$post_type = register_post_type( $this->post_type );
```